CONTRIBUTING.md

```diff
@@ -36,14 +36,20 @@
 Each notebook should begin with a "Learning objectives" section.
 This section contains a compact summary of the goals for the notebook, in the form of a bulleted list of objectives, introduced by the phrase:
 
-> This notebook will demonstrate how to
+> This notebook will demonstrate how to:
```

Comment:
Can we get guidance on the `## Objectives` header, too? That appears to be a thing we are doing consistently across two PRs!

Reply:
I just copied @allyhawkins, but it made sense to me!

Reply:
I added in guidelines for the title of the header and spacing between lines for the header into the CONTRIBUTING.md doc to hopefully clarify.